lib/shoulda/matchers/action_controller/permit_matcher.rb

```diff
@@ -250,7 +250,10 @@ def matches?(controller)
           parameters_double_registry.register
 
           Doublespeak.with_doubles_activated do
-            context.__send__(verb, action, request_params)
+            scoped = ::ActionPack::VERSION::MAJOR >= 5
```

Comment:
Thanks for this. We tend to place logic that's dependent on Rails versions in RailsShim, that way everything is in one place. What if you extracted `context.__send__` to a method, say, `make_controller_request(context, verb, action, options = {})`?

Additionally, please note that I won't be able to merge this in until